app/controllers/tenants/projects_controller.rb

```diff
@@ -18,15 +18,30 @@ def show
   end
 
   def create
-    @project = Project.create(project_params)
-    add_category(params[:project][:categories])
-    redirect_to projects_path
+    @project = Project.create!(title: project_params[:title],
+                               description: project_params[:description],
+                               price: project_params[:price],
+                               tenant_id: current_user.tenant_id,
+                               categories: new_categories)
+    redirect_to tenant_dashboard_path
```

Comment:
Done. Still could use some work; it would be especially great if we could figure out how to get the categories to come in with out the empty string.